### DISCUSSION TITLE
Dynamic configuration

### DISCUSSION THREAD

**Discussion by sam-cgr** [ory/kratos#2458] [Ideas]
I was wondering if there is anyway to hot load / modify settings at runtime from a source other than file, such as api call or database. This would make sense for things like OIDC configurations so you could add or change one without a full server reboot. Maybe this has been covered before.

**Comment by vinckr** [comment]
Hey Sam, 
I am not sure how it works but in the managed Ory Cloud you can just add/remove/configure OIDC providers without downtime, and since its running Ory Kratos in the background its already possible. 
But we dont have any documentation for it afaik 🤔
